phpunit.xml.dist

```diff
@@ -13,8 +13,7 @@
 
     <filter>
         <whitelist>
-            <file suffix=".php">./src/NexyCrypt.php</file>
-            <file suffix=".php">./src/PrivateKey.php</file>
+            <directory suffix=".php">./src/</directory>
```

Comment:
No need for `suffix` key.

Please use this file: https://github.com/nexylan/slack-bundle/blob/master/phpunit.xml.dist

We have the same for all Nexylan's projects. :wink: